plugins/textpattern.md

```diff
@@ -31,8 +31,9 @@ This option allows configuring the text patterns that get matched by the `textpa
 
 There are three types of patterns: `inline`, `block`, and `replacement` patterns. Inline patterns have a `start` and an `end` text pattern whereas the block and replacement patterns only have a `start`. A user can specify the formats to be applied to the selection, commands to be executed, or text to be replaced.
 
-When using list commands make sure that the [lists plugin]({{ site.baseurl }}/plugins/lists) is registered for normalized behavior across browsers.
+> **Note**: When using list commands make sure that the [lists plugin]({{ site.baseurl }}/plugins/lists) is registered for normalized behavior across browsers.
 
+&nbsp;
```

Comment:
I think this may have been added accidentally?

Reply:
Nope, its actually to give a new line so the two notes don't get concatenated in one. I'm not sure of a better solution :-(

Reply:
Done as bellow comment.

Reply:
I realise this has changed anyway, but for future reference I'm pretty sure you just do a newline in markdown to split them, no need for the `&nbsp;`
```
> **Note**: Some text

> **Note**: Some text in a new block
```

Reply:
It won't work @JamesToohey  :-). Me and Tyler tried that. It concatenates them. It may be the markdown linter that we use that auto-fix things when I save the file. It gets rid of the extra line, so one sits right beside the other which 'concatenate" them :-)